ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -430,6 +434,11 @@ class GraphsContainer extends React.Component {
               />
               <VictoryAxis
                 tickCount={6}
+                tickFormat={x =>
```

Comment:
tickFormat is now listed twice due to changes someone else made, so your function is not being applied (the set tickFormat is overriding it).